src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1183,7 +1183,12 @@ GlobeControls.prototype.setCameraTargetGeoPosition = function setCameraTargetGeo
  */
 GlobeControls.prototype.lookAtCoordinate = function _lookAtCoordinate(params = {}, isAnimated = this.isAnimationEnabled()) {
     if (params.zoom) {
-        params.range = computeDistanceCameraFromTileZoom(params.zoom, this._view);
+        const delta = Math.PI / Math.pow(2, params.zoom);
```

Comment:
Could you move `computeDistanceCameraFromTileZoom(zoom, camera)` in `GlobeLayer`